integrations/datadog/.port/spec.yaml

```diff
@@ -0,0 +1,29 @@
+type: datadog
+description: Datadog integration for Port Ocean
+icon: Datadog
+features:
+  - type: exporter
+    section: APM & Alerting
+    resources:
+      - kind: datadogHost
+      - kind: datadogMonitor
+      - kind: datadogServiceCatalog
+      - kind: datadogSlo
+configurations:
+  - name: datadogBaseUrl
+    type: url
```

Comment:
Since there is no default datadog base URL, can you make this param required=true?